_config.yml

```diff
@@ -77,9 +77,12 @@ lastModified:
 blogPosts:
   en: Blog
   fr: Blogue
-notice:
```

Comment:
We need to keep this notice. We don't want to mislead the public. Only when the CIO approves our work does it become official.
I'd even say this notice needs to be at the top of every page.
The notice should say that once something is "approved" or "official" that it is found _add link to where it should be (like Sharepoint or the official ESDC public page)_

Reply:
The notice is still there.  In the footer next to View on Github, the Notice link opens a modal (popup) with the notice.

Is that good?